### PR TITLE
[RM] changelog for 8.13.2

### PR DESCRIPTION
**Pull request by gares**

### CODE REVIEW

File: doc/sphinx/changes.rst [hunk 1]
```diff
@@ -721,6 +721,33 @@ CoqIDE
...
+  by Guillaume Melquiond).
+- **Fixed:**
+  Never store persistent arrays as VM / native structured values.
+  This could be used to make vo marshalling crash, and probably
+  breaking some other invariants of the kernel
+  (`#14007 <https://github.com/coq/coq/pull/14007>`_,
+  fixes `#14006 <https://github.com/coq/coq/issues/14006>`_,
+  by Pierre-Marie Pédrot).
+
+Tactics
```
Comment (Zimmi48) [inline doc/sphinx/changes.rst:743]: I think it was probably a mistake to put the changelog entry about Ltac2 in "Tactics":
```suggestion
Tactic language
```

File: doc/sphinx/changes.rst [hunk 2]
```diff
@@ -721,6 +721,33 @@ CoqIDE
...
+- **Fixed:**
+  Never store persistent arrays as VM / native structured values.
+  This could be used to make vo marshalling crash, and probably
+  breaking some other invariants of the kernel
+  (`#14007 <https://github.com/coq/coq/pull/14007>`_,
+  fixes `#14006 <https://github.com/coq/coq/issues/14006>`_,
+  by Pierre-Marie Pédrot).
+
+Tactic language
+^^^^^^^
```
Comment (Zimmi48) [inline doc/sphinx/changes.rst:744]: I forgot about this:

```suggestion
^^^^^^^^^^^^^^^^
```

**Comment by Zimmi48** [comment]
@gares If you end up not backporting #14012, it should be removed from here. So tell me when you are sure that it will be backported so that I can merge.

**Comment by gares** [comment]
merge plz

**Review by Zimmi48 — APPROVED** [review]
@coqbot merge now

**Pull request merged**